src/mainnet/target-assets/stETHAssetVault.sol

```diff
@@ -0,0 +1,128 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+// ███████╗░█████╗░██████╗░████████╗██████╗░███████╗░██████╗░██████╗
+// ██╔════╝██╔══██╗██╔══██╗╚══██╔══╝██╔══██╗██╔════╝██╔════╝██╔════╝
+// █████╗░░██║░░██║██████╔╝░░░██║░░░██████╔╝█████╗░░╚█████╗░╚█████╗░
+// ██╔══╝░░██║░░██║██╔══██╗░░░██║░░░██╔══██╗██╔══╝░░░╚═══██╗░╚═══██╗
+// ██║░░░░░╚█████╔╝██║░░██║░░░██║░░░██║░░██║███████╗██████╔╝██████╔╝
+// ╚═╝░░░░░░╚════╝░╚═╝░░╚═╝░░░╚═╝░░░╚═╝░░╚═╝╚══════╝╚═════╝░╚═════╝░
+// ███████╗██╗███╗░░██╗░█████╗░███╗░░██╗░█████╗░███████╗
+// ██╔════╝██║████╗░██║██╔══██╗████╗░██║██╔══██╗██╔════╝
+// █████╗░░██║██╔██╗██║███████║██╔██╗██║██║░░╚═╝█████╗░░
+// ██╔══╝░░██║██║╚████║██╔══██║██║╚████║██║░░██╗██╔══╝░░
+// ██║░░░░░██║██║░╚███║██║░░██║██║░╚███║╚█████╔╝███████╗
+// ╚═╝░░░░░╚═╝╚═╝░░╚══╝╚═╝░░╚═╝╚═╝░░╚══╝░╚════╝░╚══════╝
+
+
+// Github - https://github.com/FortressFinance
+
+import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {ReentrancyGuard} from "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+import {ISTETH, IWSTETH} from "./interfaces/ISTETH.sol";
+import {IMetaVault} from "./interfaces/IMetaVault.sol";
+import {IAssetVault} from "./interfaces/IAssetVault.sol";
+
+contract StETHAssetVault is ReentrancyGuard, IAssetVault {
+
+    using SafeERC20 for ERC20;
+
+    /// @notice The metaVault that manages this vault
+    address public metaVault;
+    /// @notice The platform address
+    address public platform;
+    /// @notice Enables Platform to override isStrategiesActive value
+    bool public isActiveOverride = true;
+
+    /// @notice The address of WETH token.
+    address internal constant WETH = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+    /// @notice The address of WSTETH token.
+    address internal constant WSTETH = 0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0;
+    /// @notice The address of STETH token.
+    address internal constant STETH = 0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84;
```

Comment:
How do we add or remove LSDs when these are hard coded here?

Reply:
LSD vaults are added/removed on fortETH contract. The commented one is stETH LSD vault implementation.